specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/common.json

```diff
@@ -67,7 +67,8 @@
           "$ref": "#/definitions/Error"
         }
       },
-      "description": "The key vault server error."
+      "description": "The key vault server error.",
+      "x-nullable": true
```

Comment:
@johanste is this correct?  I thought `x-nullable` would be applied where this type is returned.

Reply:
Does the service actually return `null` here? If so, why? The property is optional. And we really don't want to have semantically meaningful `null` values for content-type application/json payloads.

Reply:
In my testing the service returns null for properties of the `Error` model, including `innererror`, which is of type `Error`.

https://github.com/Azure/azure-rest-api-specs/blob/625a4165f8a2ee84549f61e46620e1442a509ab5/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/common.json#L64-L68

Reply:
If the intention is to indicate that fields of the `Error` type can be null then you want to apply `x-nullable` to the applicable fields (like you did in backuprestore.json).

Reply:
The error value itself is returned as null. I think the service should clarify which of the types and properties are  expected to be nullable before finalizing on the correct swagger change. 

Currently, marking just the Error properties results in a null ref exception in generated code, but marking the whole Error type does not.

Reply:
Did you check with @pakrym - or even just try it with local swaggers - if setting `x-nullable: true` on the model itself even works as intended?

Reply:
Yes to both :)

Reply:
Managed HSM usage of Error is that when there is no error in a correct case we are returning null to indicate the same.

Reply:
The service should not return a `null` value on the wire. You should omit the key/value pair altogether. Is there a reason that the service is sending null on the wire?

Reply:
In speaking with @vasanthrajams previously, this is consistent with existing Key Vault api behavior, so changing it now could be breaking.